### PR TITLE
add best youtuber for learning  code

### PR DESCRIPTION
**Pull request by A3RC**
Just added Info about clear code and super simple dev

### CODE REVIEW

File: content/contributing.md
Comment (crebro) [inline content/contributing.md:1]: why this change?

Comment (A3RC) [reply content/contributing.md:1]: any problem appearing??

Comment (crebro) [reply content/contributing.md:1]: you are editing `content/contribuiting.md` which lies outside the scope of your commit.

Comment (A3RC) [reply content/contributing.md:1]: ok but i believe i didn't change anything
